acct.bionimbus.org/portal/gitops.json

```diff
@@ -79,253 +93,329 @@
         {
           "icon": "dictionary",
           "link": "/DD",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "Dictionary"
         },
         {
           "icon": "exploration",
-          "link": "/files",
-          "color": "#a2a2a2",
+          "link": "/explorer",
+          "color": "#1a3057",
           "name": "Exploration"
         },
         {
           "icon": "query",
           "link": "/query",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "query"
         },
         {
           "icon": "workspace",
-          "link": "/workspace",
-          "color": "#a2a2a2",
+          "link": "#hostname#workspace/",
```

Comment:
Nah. Needs to be the prod-version

Reply:
Again, the automation is supposed to copy whatever it finds on gitops-qa/qa-acct.
Did some bad copy happen here? Or did it ignore some json blocks from gitops-qa/qa-acct ?